README.md

```diff
@@ -23,7 +23,7 @@ Shopify’s TSLint rules come bundled in `tslint-config-shopify`.
 To enable these rules, create a `tslint.json` file at the root level of your project, and extend `tslint-config-shopify`.
 ```
 {
-  "extends": "tslint-config-shopify"
+  "extends": "tslint-config-shopify/base"
```

Comment:
Awesome, thanks for that list of rules 👍. 
Did you narrow down that list manually?